go/oasis-test-runner/scenario/e2e/runtime/sentry.go

```diff
@@ -139,11 +148,14 @@ func (s *sentryImpl) Fixture() (*oasis.NetworkFixture, error) {
 			// Disable cert rotation on one of the storage nodes so we can use
 			// its TLS certificates in the access control sanity checks.
 			DisableCertRotation: true,
+			// Also disable public RPC on one, so we can check access control.
+			DisablePublicRPC: true,
 		},
 		{
-			Backend:  database.BackendNameBadgerDB,
-			Entity:   1,
-			Sentries: []int{4},
+			Backend:             database.BackendNameBadgerDB,
+			Entity:              1,
+			Sentries:            []int{4},
+			DisableCertRotation: true,
```

Comment:
This disables cert rotation on both storage workers, so this part is not tested?